tests/ui/fn/ok.stderr

```diff
@@ -0,0 +1,29 @@
+Error: expected `<`
+
+Caused by:
+    0: () {}
+       
+               fn with_return() -> &str {}
+       
+               fn is_true(v: bool) -> bool {}
+       
+               fn add(x: i64, y: i64) -> i64 {}
+       
+               fn return_multi(a: i32, b: i32) -> (i32, i32) {}
+           }
+       ]
+       
+    1: failed to parse [
```

Comment:
This is a parser error. Have you looked at the other tests that demonstrate the difference in a-mir-formality syntax?